frontend/packages/ux-editor/src/components/config/editModal/EditImage/LocalImage/ImportImage/ImportImage.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { renderWithProviders } from '@altinn/ux-editor/testing/mocks';
+import { ImportImage } from '@altinn/ux-editor/components/config/editModal/EditImage/LocalImage/ImportImage/ImportImage';
```

Comment:
Jeg trodde vi hadde linting på dette, men vi bruker i hvert fall relative filstier for ting som er i samme pakke.
```suggestion
import { renderWithProviders } from '../../../../../../testing/mocks';
import { ImportImage } from './ImportImage';
```